search/ping.py

```diff
@@ -0,0 +1,13 @@
+from flask import Blueprint, Response
+
+
+def get_ping_blueprint() -> Blueprint:
+    blueprint = Blueprint('ping', __name__)
+
+    @blueprint.route('/ping', methods=['GET'])
+    def search() -> Response:  # pylint: disable=unused-variable
+        response = Response(response='pong', status=200, mimetype='text/plain')
+        response.headers['Cache-Control'] = 'no-store, must-revalidate'
```

Comment:
Need to also follow `It MUST return a Expires header with the value 0 if HTTP/1.0 clients/proxies are used`

Reply:
updated.